android/titanium/src/java/org/appcelerator/titanium/util/TiBlobLruCache.java

```diff
@@ -37,10 +37,6 @@ public TiBlobLruCache()
 	@Override
 	protected int sizeOf(String key, Bitmap bitmap)
```

Comment:
We can rather do this:
```
@Override
protected int sizeOf(String key, Bitmap bitmap)
{
	return bitmap.getAllocationByteCount();
}
```
1. Considers bitmap memory reuse across decode/encode operations.
2. Gives the actual memory footprint of a Bitmap in RAM.
3. Already returns `0` for recycled bitmaps.